rpisense-core.c

```diff
@@ -20,37 +20,24 @@
 #include <linux/init.h>
 #include <linux/i2c.h>
 #include <linux/platform_device.h>
-#include "core.h"
+#include "rpisense.h"
 #include <linux/slab.h>
 
-static void rpisense_client_dev_register(struct rpisense *rpisense,
-					 const char *name,
-					 struct platform_device **pdev)
-{
-	int ret;
+#define RPISENSE_FB			0x00
+#define RPISENSE_WAI			0xF0
+#define RPISENSE_VER			0xF1
+#define RPISENSE_KEYS			0xF2
+#define RPISENSE_EE_WP			0xF3
 
```

Comment:
what is the reasoning moving these from the header file into core.c?

Reply:
wanted to hide the details of i2c from the public header. This change makes more sense in context of what I did for the frame buffer. I actually moved a lot of the code for refreshing the display  into the function that replaces block write.